.changeset/dull-carpets-breathe.md

```diff
@@ -21,10 +21,10 @@ import ReactComponent from "../components/button.jsx"
 import VueComponent from "../components/button.vue"
 
 // MDX runtime is contained inside the Astro
```

Comment:
@ematipico this sentence feels incomplete, inside the Astro what?

Reply:
It should be 
> // MDX runtime is contained inside Astro

Reply:
> // MDX runtime is contained inside the Astro core

I use it.